packages/web-twig/src/Resources/components/Item/README.md

```diff
@@ -1,8 +1,87 @@
-# Item
+#Item
 
-To implement Item component with content of `Radio` or `Checkbox`,
-you need to use directly these components with prop `isItem`.
+The Item component is used to display a single item in a list. It can be used in Dropdown or similar.
 
-See `web` package [Item documentation][item-documentation] for more info.
+To use Item with checkbox or radio please use components [Checkbox][checkbox] or [Radio][radio]
+with `isItem` property. We do this to avoid repeating the same code and to simplify the API.
 
-[item-documentation]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/Item/README.md
+Simple Item example:
+
+```twig
+<Item label="Item" />;
+```
+
+Item with icon example:
+
+```twig
+<Item label="Item" iconName="search" />;
+```
+
+Item in selected state example:
+
+```twig
+<Item label="Item" isSelected />;
+```
+
+Item with Helper text example:
+
+```twig
+<Item label="Item" helperText="Helper text" />;
+```
+
+Item in disabled state example:
+
+```twig
+<Item label="Item" isDisabled />;
+```
+
+Item with icon and helper text in selected state example:
+
+```twig
+<Item label="Item" iconName="search" helperText="Helper text" isSelected />;
+```
+
+Item as a link example:
+
+```twig
+<Item label="Item" elementType="a" href="#" />;
+```
+
+Radio as Item:
+
+```twig
+<Radio id="radioItem" name="example" label="Radio Label" isItem />;
+```
+
+Checkbox as Item:
+
+```twig
+<Checkbox id="checkboxItem" name="example" label="Checkbox Label" isItem />;
+```
+
+## API
```

Comment:
We should mention the href etc too. It can be in the table or somehow below.

Reply:
href, target and type added to API table